TUTORIALS.md

```diff
@@ -1,98 +1,86 @@
 # TUTORIALS
 
-## Install and build Adlik
+## Simple example by using provided docker
 
-Follow the [Usage](adlik_serving/README.md#Usage) compile your model.
+This example demonstrates how to use Adlik to compile serving model and deploy serving service end to end.
 
->Note:
->You need to build the Adlik serving accord to the demand scenario and environment hardware configuration. If you want
->to build TensorRT or TensorFlow GPU service, you need to make sure that your environment has the GPU and the CUDA
->driver vision >= 410.48 (click [here](https://docs.nvidia.com/cuda/cuda-toolkit-release-notes/index.html) for details).
+### Compile the model
 
-## Optimize model
+1. Pull the compiler docker image
 
-Follow the [Usage](https://github.com/Adlik/model_optimizer/blob/master/README.md#usage) optimize your model.
-
-## Compile model
-
-Follow the [Usage](model_compiler/README.md#usage) compile your model.
+    ```sh
+    docker pull registry.cn-beijing.aliyuncs.com/adlik/model-compiler:7.0.0.11_10.0
+    ```
 
-> Note:
->
-> You need to pay attention that the compiled model type needs to be consistent with the build serving type.
->
-> When you compile the TensorRT model, the version of the TensorRT python package must be the same as the version of
-> TensorRT (`dpkg -l | grep TensorRT`)
+2. Train a model and save as .h5 file:
 
-## Deploy serving service
+    ```sh
+    git clone https://github.com/Adlik/Adlik.git
 
-After you build the serving and compile the model, you can deploy the serving service.
+    cd Adlik/examples/keras_model
+    
+    python3 train_model.py
+    ```
 
-### OpenVINO service
+   After running this script, there is a .h5 file in the “model” directory.
 
-```sh
-source /opt/intel/openvino_VERSION/bin/setupvars.sh
-cd {dir_of_adlik_serving_binary}
-./adlik_serving --model_base_path={model_repos_dir} --grpc_port={grpc_port} --http_port={http_port}
-```
+3. Run the image
 
-### Other service
+   In the `Adlik/examples/keras_model` directory, run the command:
```

Comment:
You have already `cd Adlik/examples/keras_model`, so you can start the image directly:

```suggestion
   In the `Adlik/examples/keras_model` directory, run the command:
```
```suggestion
   Run the following command:
```